src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -51,6 +51,9 @@ function DatasetOverviewPanel(props) {
   const { classes, dataset, dispatch, userRoles } = props;
   const linkToBq = dataset.accessInformation?.bigQuery !== undefined;
 
+  const canManageUsers =
+    userRoles.includes(DatasetRoles.STEWARD) || userRoles.includes(DatasetRoles.CUSTODIAN);
```

Comment:
I don't think custodians can add stewards so we may need to have more targeted checking

Reply:
Added a variable for `canManageStewards`!

Reply:
This section will continue to just use `canManageUsers` because it is the logic to show/hide the tab on the overview page, and we do a separate check again later inside datasetAccess to see which ones they have permission to manage